src/frontend/typer/behandling.ts

```diff
@@ -137,6 +137,7 @@ export const hentStegNummer = (steg: BehandlingSteg): number => {
 export enum BehandlingStatus {
     OPPRETTET = 'OPPRETTET',
```

Comment:
Denne ble fjernet i BA-sak sin PR, men antar at den kanskje skal brukes her da

Reply:
Tenkte det var greit å beholde for sikkerhetsskyld, så lenge det er en verdi som eksisterer for BehandlingStatus i backend.